centreon/src/App/Security/Infrastructure/Idp/LocalIdp.php

```
@@ -0,0 +1,35 @@
+ */
+
+declare(strict_types=1);
+
+namespace App\Security\Infrastructure\Idp;
+
+use App\Security\Domain\Aggregate\Token;
+use App\Security\Domain\Exception\TokenRefreshUnavailableException;
+
+final readonly class LocalIdp implements IdpInterface
+{
+    public function refreshToken(Token $token): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$token'. (undefined)

(UnusedFormalParameter)